lib/basic-auth-middleware.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
```

Comment:
Great work refactoring the basic and bearer auth middleware to support your store model